# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

UserTrackingActionGoal action_goal
UserTrackingActionResult action_result
UserTrackingActionFeedback action_feedback
